tests/e2e/specs/stories-editor/page.js

```diff
@@ -218,4 +248,102 @@ describe( 'Story Page', () => {
+
+		await page.click( storySettingsPanelSelector );
+		await page.waitForSelector( storySettingsAdvanceAfterSelector );
+
+		await page.select( storySettingsAdvanceAfterSelector, 'auto' );
+
+		await saveDraft();
+		await page.reload();
+
+		await page.click( documentPanelSelector );
+		await page.waitForSelector( storySettingsPanelSelector );
+
+		await page.click( storySettingsPanelSelector );
+		await page.waitForSelector( storySettingsAdvanceAfterSelector );
+
+		expect( await page.evaluate( ( selector ) => {
+			return document.querySelector( selector ).value;
+		}, storySettingsAdvanceAfterSelector ) ).toBe( 'auto' );
+	} );
+
+	it( 'should apply story settings to newly created pages', async () => {
+		await page.click( '.block-editor-inserter .editor-inserter__amp-inserter' );
+		await page.waitForXPath( `//div[contains(@class, 'amp-story-page-number') and contains(text(), 'Page 2')]` );
+
+		const pageAdvancementSelector = '.components-select-control__input';
+		await page.waitForSelector( pageAdvancementSelector );
+
+		const secondsSelector = 'input[aria-label="Time in seconds"]';
+		await page.waitForSelector( secondsSelector );
+
+		expect( await page.evaluate( ( selector ) => {
+			return document.querySelector( selector ).value;
+		}, pageAdvancementSelector ) ).toBe( 'time' );
+
+		expect( await page.evaluate( ( selector ) => {
+			return document.querySelector( selector ).value;
+		}, secondsSelector ) ).toBe( '2' );
+	} );
+
+	it( 'should not affect story settings when individual page settings are changed', async () => {
+		await page.click( '.block-editor-inserter .editor-inserter__amp-inserter' );
+		await page.waitForXPath( `//div[contains(@class, 'amp-story-page-number') and contains(text(), 'Page 2')]` );
+
+		const pageAdvancementSelector = '.components-select-control__input';
+		await page.waitForSelector( pageAdvancementSelector );
+
+		const secondsSelector = 'input[aria-label="Time in seconds"]';
+		await page.waitForSelector( secondsSelector );
+
+		await page.evaluate( ( selector ) => {
+			document.querySelector( selector ).value = '';
+		}, secondsSelector );
+
+		await page.type( secondsSelector, '5' );
+
+		const documentPanelSelector = 'button.edit-post-sidebar__panel-tab[data-label="Document"]';
+		const storySettingsPanelSelector = '.amp-story-settings-panel';
+		const storySettingsAdvanceAfterDurationXPathSelector = `//div[contains(@class, 'components-panel__body') and contains(@class, 'amp-story-settings-panel')]//input[contains(@class, 'components-range-control__number')][1]`;
+
+		await page.click( documentPanelSelector );
+		await page.waitForSelector( storySettingsPanelSelector );
+
+		await page.click( storySettingsPanelSelector );
+		const durationSettingHandle = await page.waitForXPath( storySettingsAdvanceAfterDurationXPathSelector );
+
+		expect( await page.evaluate( ( node ) => {
+			return node.value;
+		}, durationSettingHandle ) ).toBe( '2' );
+	} );
```

Comment:
There should be a test for following. 

Set defaults. 
Create story. 
Save story. 
check values. 
Set defaults to something new. 
Create story. 
Save story. 
check values. 
Load original story. 
Check values.

Reply:
https://github.com/ampproject/amp-wp/pull/3201#issuecomment-531178001